File: Lv.2/70129.js
function solution(s) {
    let count = 0;
    let deleted_zero = 0;
    
    while (s.length != 1) {
        previous = s.length;
        s = s.replace(/0/g, '');
        current = s.length
        s = s.length.toString(2);
        count++;
        deleted_zero += previous - current;
    }
    
    return [count, deleted_zero];
}

File: Lv.2/70129.py
def solution(s):
    count = 0
    deleted_zero = 0
    
    while (len(s) != 1):
        previous = len(s)
        s = s.replace('0', '')
        current = len(s)
        s = str(format(len(s), 'b'))
        count += 1
        deleted_zero += previous - current
        
    return [count, deleted_zero]
